serenity-core/src/test/groovy/net/serenitybdd/core/photography/integration/WhenAPhotographerTakesLotsOfScreenshots.groovy

```diff
@@ -27,7 +27,9 @@ class WhenAPhotographerTakesLotsOfScreenshots extends Specification {
             Darkroom.waitUntilClose();
             driver.quit()
         then:
-            photos.each { photo -> Files.exists(photo.pathToScreenshot) }
+            photos.findAll {
+                photo -> Files.exists(photo.pathToScreenshot)
+            }.size() == 11
```

Comment:
Yes, this is size of collection, but of filtered collection. So I take all photos, and filter them by parameter "exists", and after that check if size of result collection correct. 

Here is short reference about how this works

http://grails.asia/groovy-findall